SPECS/llvm/llvm.spec

```diff
@@ -99,6 +104,9 @@ ninja check-all
 %{_includedir}/*
 
 %changelog
+* Fri Apr 5 2024 Mykhailo Bykhovtsev <mbykhovtsev@microsoft.com> - 18.1.2-2
+- Added LLVMgold.so files to the main package
```

Comment:
Might need to tweak the changelog a bit, since we only added a symbolic link - the library has been there for a while.